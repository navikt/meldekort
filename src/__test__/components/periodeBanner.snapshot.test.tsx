import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ProviderWrapper from '../utils/providerWrapper';
import PeriodeBanner from '../../app/components/periodeBanner/periodeBanner';

it('Sprakvelger', () => {
  const tree = renderer.create(
    <ProviderWrapper>
      <PeriodeBanner />
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
