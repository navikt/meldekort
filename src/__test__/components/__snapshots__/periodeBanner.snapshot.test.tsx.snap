// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sprakvelger 1`] = `
<section
  className="seksjon periodeBanner "
>
  <p
    className="typo-ingress flex-innhold sentrert"
  >
    <span>
      Meldekort for perioden 
    </span>
  </p>
  <h1
    className="typo-innholdstittel flex-innhold sentrert"
  >
    Uke  28 - 28
  </h1>
  <p
    className="typo-normal flex-innhold sentrert"
  >
    10.07.2019 - 10.07.2019
  </p>
</section>
`;
